scripts/upgrade-vertica.sh

```diff
@@ -123,9 +123,21 @@ kubectl patch vdb $VDB_NAME --type=merge -p '{"spec": {"autoRestartVertica": fal
 kubectl wait --for=condition=AutoRestartVertica=False vdb/$VDB_NAME --timeout="${TIMEOUT}"s $NS_OPT 1> /dev/null
 echo "autoRestartVertica successfully disabled"
 
-echo "starting stop_db..."
-kubectl exec $NS_OPT $POD_NAME -- admintools -t stop_db -F -d $VDB_DBNAME $PS_OPT 1> /dev/null
-echo "cluster down"
+RUNNING_PODS=$(${GET_POD} --field-selector=status.phase=Running 2> /dev/null | grep ${VDB_NAME} | wc -l)
```

Comment:
$POD_NAME is a pod that we'll use to call AT.  But we don't verify that it is running.  Should we put the `--field-selector` argument in $POD_NAME and use that?  We don't need to check that all pods are running.  Some may not be running but the db could still be up.  If we have one pod running, that should be enough to check if the cluster is up with `list_allnodes`

Reply:
Ok got it